app.rb

```diff
@@ -0,0 +1,28 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require 'sinatra/partial'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+    register Sinatra::Partial
+  end
+
+  enable :sessions
+
+  get '/' do
+    @peeps = Peep.all
+    @peep_time = session[:time]
+    p @peep_time
```

Comment:
If this was for debugging, it's better practice to take it out before opening a PR.